cpp/ql/src/semmle/code/cpp/security/CommandExecution.qll

```diff
@@ -137,8 +137,8 @@ class ArrayExecFunctionCall extends FunctionCall {
    * all the other ones start with the command. */
   private int getCommandIdx() {
     if (
-      getTarget().getQualifiedName().matches("\\_spawn%")
-      or getTarget().getQualifiedName().matches("\\_wspawn%"))
+      getTarget().getName().matches("\\_spawn%")
+      or getTarget().getName().matches("\\_wspawn%"))
```

Comment:
Same here

Reply:
I think it's fine, for the same reason as https://github.com/Semmle/ql/pull/1303#discussion_r280804696, but please double-check.